lua/lspconfig/jsonnet_ls.lua

```diff
@@ -1,9 +1,19 @@
 local configs = require 'lspconfig/configs'
 local util = require 'lspconfig/util'
 
+-- common jsonnet library paths
+local function jsonnet_path()
+  local paths = {
+    vim.fn.getcwd() .. '/lib',
+    vim.fn.getcwd() .. '/vendor',
```

Comment:
Shouldn't this be relative to the resolved `root_dir` rather than `cwd`?

Reply:
Also use path.join instead of string concatenation.